services/order-service.js

```diff
@@ -32,9 +32,17 @@ exports.getAllOrders = async function () {
         let orders = await Transaction.findAll({
             where: {
                 admin_id: "ORDER",
-                status: { [Op.or]: ["pending", "inUse"] }
+                status: { [Op.or]: ["pending", "inUse", "late"] }
             }
         });
+
+        // update status on any late orders
+        orders.forEach((trans) => {
```

Comment:
I think you should add this check/update to `transaction-service.js` as well (`getAllTransactions` and `getUserPurchaseHistory`). So if the user pulls their own transaction history, we also check and update late ones.